backend/webapi/Features/CommonHandlers/UpdateBcProviderAttributesConsumer.cs

```diff
@@ -43,4 +43,3 @@ public static partial class UpdateBcProviderAttributesConsumerLoggingExtensions
     [LoggerMessage(1, LogLevel.Error, "Error when updating attributes to User {upn} in Azure AD.")]
     public static partial void LogUpdateBcProviderAttributesFailed(this ILogger<UpdateBcProviderAttributesConsumer> logger, string upn);
 }
-
```

Comment:
We generally leave a trailing newline character in our files; no need to remove this.